gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -959,6 +960,50 @@ impl GatewayLightning for ClnRpcService {
         }
     }
 
+    async fn withdraw_onchain(
+        &self,
+        request: tonic::Request<WithdrawOnchainRequest>,
+    ) -> Result<tonic::Response<WithdrawOnchainResponse>, Status> {
+        let request_inner = request.into_inner();
+
+        let txid = self
+            .rpc_client()
+            .await
+            .map_err(|e| Status::internal(e.to_string()))?
+            .call(cln_rpc::Request::Withdraw(
+                model::requests::WithdrawRequest {
+                    feerate: Some(cln_rpc::primitives::Feerate::PerKw(
+                        // 1 vbyte = 4 weight units, so 250 vbytes = 1,000 weight units.
+                        request_inner.fee_rate_sats_per_vbyte as u32 * 250,
+                    )),
+                    minconf: Some(0),
```

Comment:
Given that you're setting the feerate above, should this be `None`?

Reply:
This doesn't relate to the feerate. This determines UTXO selection for the sweep. I set this to 0 so that users can sweep funds of channels they just closed.

Reply:
Ah ok I misread the docs. Yeah 0 is the correct value here IMO